packages/v4/src/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -0,0 +1,43 @@
+---
+id: Helper text 
+section: components
+---
+
+**Helper text** is an on-screen field guideline that helps provide context regarding field inputs. 
+<br/>
+## Usage
+Helper text is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appear as a subheading, unlike [popovers](https://www.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
+<br/>
+### When to use helper text
+* Use helper text to provide imperative information regarding an input, such as character limits, password expectations, and more.  
+* Use colored helper text to convey the status of a user's input. For example, if a user has met the character requirements, display helper text in green font to show success. Similarly, if a user has not met a password requirement, display the helper text in red to show failure/error. <br/>
+<img src="./img/errorhelptext.png" alt="image showing error help text" width="452"/>
+<img src="./img/coloredhelptext.png" alt="image showing error help text" width="452"/>
```

Comment:
Just realized the colors used in this image aren't correct- just updated it in Sketch to the right colors, so just need to re-export and replaced!